### ISSUE TITLE
NERSC config fails to compile gmp due to libtool version incompatibility

### ISSUE THREAD

**Issue by bcfriesen** [spack/spack-configs#10]
When using the NERSC `packages.yaml` file in this repo, `spack install flang` on Cori fails during compilation of gmp:

```console
==> Installing gmp
==> Searching for binary cache of gmp
==> No binary for gmp found: installing from source
==> Using cached archive: /global/gscratch1/sd/friesen/spack/cache/gmp/gmp-6.1.2.tar.bz2
==> Staging archive: /global/u2/f/friesen/spack/var/spack/stage/gmp-6.1.2-c36gbbzmkcbaythbgglvr5tl2xccsf2m/gmp-6.1.2.tar.bz2
==> Created stage in /global/u2/f/friesen/spack/var/spack/stage/gmp-6.1.2-c36gbbzmkcbaythbgglvr5tl2xccsf2m
==> No patches needed for gmp
==> Building gmp [AutotoolsPackage]
==> Executing phase: 'autoreconf'
==> Executing phase: 'configure'
==> Executing phase: 'build'
==> Error: ProcessError: Command exited with status 2:
    'make' '-j64'

66 errors found in build log:
     693     libtool: and run autoconf again.
     694     libtool: Version mismatch error.  This is libtool 2.4.6, but the
     695     libtool: definition of this LT_INIT comes from libtool 2.4.2.
     696     libtool: You should recreate aclocal.m4 with macros from libtool 2
             .4.6
     697     libtool: and run autoconf again.
     698     Makefile:491: recipe for target 'mp_bases.lo' failed
```

This error does not occur if I do not use this `packages.yaml`.

**Comment by mic84** [comment]
I have the same exact issue when trying to install amrex via  `spack install -j8 amrex@develop  ^cmake@3.13.0` on Cori using NERSC packages.yaml:

**Comment by tgamblin** [comment]
@mamelara

**Comment by bcfriesen** [comment]
Hi, any update on this?

**Comment by mamelara** [comment]
Hi @bcfriesen sorry I've been working on finding out potential causes of the AMReX slowdown on Cori. I'll get to updating once I figure out how to resolve the issue via the config

**Comment by bcfriesen** [comment]
Got it - thanks @mamelara !

**Comment by mamelara** [comment]
Can you try my PR @bcfriesen ?

**Comment by bcfriesen** [comment]
@mamelara Your PR works. Please merge with abandon. Thanks!

**Comment by tgamblin** [comment]
Closed by #11.
